app/views/layout.erb

```diff
@@ -15,9 +15,29 @@
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
   <script src="/js/application.js"></script>
 
-  <title></title>
+  <title>Survey Squid</title>
 </head>
 <body>
+  <header>
+    <a href="/"><span class="brand">SurveySquid</span></a>
+    <nav>
+      <ul>
+        <a href="/surveys"><li class='nav-link'>My Surveys</li></a>
```

Comment:
this link should be available if a user is logged in, along with the logout link. we can use the  Logged In method that lives in the helper file: 
```ruby
if logged_in?

```